packages/core/saved-objects/core-saved-objects-migration-server-internal/src/kibana_migrator_utils.ts

```diff
@@ -67,7 +67,7 @@ export async function getCurrentIndexTypesMap({
         client.indices.getMapping({
           index: mainIndex,
         }),
-      logger,
+      undefined, // .kibana will NOT exist on a fresh start, and this should not result in a WARN in the logs
```

Comment:
Based on a code search, looks like this is the only place `migrationRetryCallCluster` is called... What do you think about just deleting logger from these params and function usage entirely?

Reply:
Sure! We can delete it